### PR TITLE
Implement Amenities

### PR DESCRIPTION
**Pull request by OzgeYurtsever**

### CODE REVIEW

File: client/src/components/Amenities/Amenities.js [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+import Icons from './Icons.js';
+
+ReactModal.setAppElement('#root');
+
+class Amenities extends Component {
+  constructor(props) {
+    super(props);
+    console.log('amenities here', this.props.amenities);
+    this.state = {
+      showPopUp: false
```
Comment (nvincenthill) [inline client/src/components/Amenities/Amenities.js]: Perhaps isPopUpDisplayed is a better name- showPopUp sounds like a function not a boolean

File: client/src/components/Amenities/Amenities.js [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+    }
+    return (
+      <div>
+        <div className="amenities section">
+          <div>Amenities</div>
+          <div className="icons--display">
+            <Icons amenities={amenitiesArr} />
+          </div>
+          <a className="toggle-more" onClick={this.handleClickShowAll}>
+            {' '}
```
Comment (nvincenthill) [inline client/src/components/Amenities/Amenities.js]: Remove unneeded space maybe?

File: client/src/components/Amenities/Icons.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+        );
+        iconCounter++;
+      } else if (
+        iconCounter > 3 &&
+        iconCounter <= 6 &&
+        current.icon.length > 0
+      ) {
+        iconsRight.push(
+          <div className={`${current.icon} icon--display__right`}>
+            {'  '}
```
Comment (nvincenthill) [inline client/src/components/Amenities/Icons.js]: Same here

File: client/src/components/Amenities/Icons.js [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+          </div>
+        );
+        iconCounter++;
+      } else if (
+        iconCounter > 3 &&
+        iconCounter <= 6 &&
+        current.icon.length > 0
+      ) {
+        iconsRight.push(
+          <div className={`${current.icon} icon--display__right`}>
```
Comment (nvincenthill) [inline client/src/components/Amenities/Icons.js]: More jQuery? What does this line do?

File: package.json [hunk 5]
```diff
@@ -38,6 +38,7 @@
     "react": "^16.4.1",
     "react-dom": "^16.4.1",
     "react-fontawesome": "^1.6.1",
+    "react-modal": "^3.5.1",
```
Comment (nvincenthill) [inline package.json]: Good choice on using this react-modal

**Review by nvincenthill — APPROVED** [review]
Nice work!

File: client/src/components/Amenities/Icons.js (hunk 4 above)
Comment (OzgeYurtsever) [reply client/src/components/Amenities/Icons.js]: It's ES6 template literals syntax.

**Pull request merged**